src/tabular-api-surface.ts

```diff
@@ -306,10 +306,17 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
       chunkTransformer = new ChunkTransformer({decode: options.decode} as any);
 
-      const reqOpts = {
-        tableName: this.name,
-        appProfileId: this.bigtable.appProfileId,
-      } as google.bigtable.v2.IReadRowsRequest;
+      const reqOpts = (
+        this.viewName
+          ? {
+              authorizedViewName: `${this.name}/authorizedViews/${this.viewName}`,
+              appProfileId: this.bigtable.appProfileId,
+            }
+          : {
+              tableName: this.name,
+              appProfileId: this.bigtable.appProfileId,
+            }
+      ) as google.bigtable.v2.IReadRowsRequest;
```

Comment:
This change makes it so that when viewName is defined, ie. when the object is an authorized view that the request will be made for an authorized view and not a table.

Reply:
Do you think a comment like this would be helpful in the code?

Reply:
Yup. Added a comment.